src/pkg/packager/publish.go

```diff
@@ -98,7 +99,8 @@ func (p *Packager) Publish() (err error) {
 			return err
 		}
 	} else {
-		if err = p.source.LoadPackage(p.layout, false); err != nil {
+		filter := &filters.EmptyFilter{}
+		if err = p.source.LoadPackage(p.layout, filter, false); err != nil {
```

Comment:
What do you think about passing in `nil` as an argument instead of a dummy implementation?

```go
LoadPackage(p.layout, nil, false)
``` 
Would have to check for `nil` before calling any methods on filter in `LoadPackage`, but seems like a clear and simple way to handle calls that don't need a filter.

Reply:
yeah, idk about using `nil` as value here, def something to discuss as a team